CONTRIBUTING.md

```diff
@@ -113,6 +113,21 @@ following rules before you submit a pull request:
    scale in dimensionality: n_features is expected to be lower than
    100".
 
+A Note about CI:
```

Comment:
OK, just to clarify, the suggestion here is to:
- revert CONTRIBUTING.md to it's original state
- change the title of the CI part as you suggest above in doc/developers/contributing.rst and also move it in that document to a more appropriate place

As for where to move it, perhaps before or after the "Testing and improving test coverage" part?

Thanks!